ui/src/ducks/app/monitoring.js

```diff
@@ -121,14 +128,31 @@ export function* fetchClusterStatus() {
     clusterHealth.apiServerStatus = getClusterQueryStatus(results[0]);
     clusterHealth.kubeSchedulerStatus = getClusterQueryStatus(results[1]);
     clusterHealth.kubeControllerManagerStatus = getClusterQueryStatus(
-      results[2]
+      results[2],
     );
     yield put(setPrometheusApiAvailable(true));
   } else {
+    const prometheusPod = yield call(
+      ApiK8s.getPodInNamespace,
+      'metalk8s-monitoring',
+      'prometheus',
+    );
+    if (!prometheusPod.error) {
+      const prometheusPodCondition =
+        prometheusPod?.body?.items[0]?.status?.conditions[0]?.type;
+      if (prometheusPodCondition === 'PodScheduled') {
+        clusterHealth.prometheusPodStatusMessage =
+          prometheusPod?.body?.items[0]?.status?.conditions[0]?.message;
+      } else {
+        console.error('This is an error in the cluster.');
+      }
+    } else {
+      console.error('Cannot find the Prometheus pod.');
```

Comment:
This could be due to another error, you should check the HTTP status code is `404` before saying that. Anyway (as said above), I don't think we should print error messages in the console.

Reply:
> This could be due to another error, you should check the HTTP status code is `404` before saying that. Anyway (as said above), I don't think we should print error messages in the console.

Removed the console.error!
Set clusterHealth.error='Unkown' instead of.